3-networks-dual-svpc/envs/production/main.tf

```diff
@@ -17,8 +17,8 @@
 locals {
   env              = "production"
   environment_code = substr(local.env, 0, 1)
-  default_region1  = "us-west1"
-  default_region2  = "us-central1"
+  default_region1  = data.terraform_remote_state.bootstrap.outputs.common_config.default_region
+  default_region2  = data.terraform_remote_state.bootstrap.outputs.common_config.default_region_2
```

Comment:
in the original version of the code default_region1 was "us-west1" and default_region2  was "us-central1".
Now default_region1 is "us-central1" and default_region2 is "us-west1".

This change will affect the integration tests because the IP ranges will change: 
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/test/integration/networks/networks_test.go#L58C1-L59C83

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/test/integration/networks/networks_test.go#L402C1-L404C173

and the mapping in other point will be affect too

https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/test/integration/networks/networks_test.go#L436C1-L449C32


The initial mapping should be restored 
OR 
the integration tests need to be updated

Reply:
Thanks! I chose to update the integration tests since the default region in 0-bootstrap was originally `us-central1`
https://github.com/terraform-google-modules/terraform-example-foundation/blob/master/0-bootstrap/variables.tf#L27-L31

Fixed in df4c103f0326c81fd25ac655c3646423d7669d1f